jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt

```diff
@@ -89,7 +89,7 @@ class CodeModernizerSession(
                 is ZipCreationResult.Succeeded -> result.payload
             }
             CodetransformTelemetry.jobCreateZipEndTime(
```

Comment:
Updated name to be more consistent across our events. This change is in vscode and `codeTransformTotalByteSize` is a "nice to have" not a business requirement.